src/iop/splittoning.c

```diff
@@ -87,7 +87,7 @@ typedef enum dt_iop_splittoning_picker_t
 
 const char *name()
 {
-  return _("split toning");
+  return _("Split Toning");
```

Comment:
All word in dt are lower case. So we cannot change that.

Reply:
I didn't know this rule.

Changing to lower case will no problem, but then I would prefer to write it split-toning as you mentioned below.

I would use the same spelling for the user manual and adjust it accordingly.

Reply:
Fine with me.

Reply:
Done!